examples/perplexity/perplexity.cpp

```diff
@@ -1097,7 +1101,7 @@ static void winogrande_score(llama_context * ctx, const common_params & params)
 
     if (params.winogrande_tasks > 0 && params.winogrande_tasks < data.size()) {
         LOG_INF("%s : selecting %zu random tasks\n", __func__, params.winogrande_tasks);
-        std::mt19937 rng(1);
+        std::mt19937 rng(std::random_device{}());
```

Comment:
I suspect that using a fixed seed was intentional so that the results are repeatable.

Reply:
I thought so too, but the rationale for the proposed change is that by "freezing" the test cases it lessens the validity of the test. If repeatable results are indeed the objective, why go through the extra trouble of randomizing in the first place, instead of just processing tests cases sequentially?

I realise this is more of a personal preference and I'm OK to revert the change back if needed.